csharp/src/Microsoft.ML.OnnxRuntime/NativeMemoryAllocator.cs

```diff
@@ -64,7 +64,7 @@ private NativeMemoryAllocatorInfo(IntPtr allocInfo)
 
         private static void Delete(IntPtr nativePtr)
         {
-            NativeMethods.ReleaseOrtAllocatorInfo(nativePtr);
+            NativeMethods.OrtReleaseAllocatorInfo(nativePtr);
```

Comment:
Would it also be beneficial to have a unit test that explicitly tests the `Delete`/`ReleaseHandle` path in the C# library?

Reply:
Will address this later. Ideally Session.Dispose() should be de-allocating, and this should be a frequently called function. Requires additional refactoring. But agreed would be good to have an explicit unit test.